src/coreclr/src/utilcode/pedecoder.cpp

```diff
@@ -2335,6 +2335,41 @@ READYTORUN_HEADER * PEDecoder::FindReadyToRunHeader() const
     return NULL;
 }
 
+uint32_t PEDecoder::GetExport(LPCSTR exportName) const
+{
+#ifndef DACCESS_COMPILE
```

Comment:
Ifdefing the body of the method with `DACCESS_COMPILE` makes it do a wrong thing for DAC build. It is not a desirable pattern that has been source of many subtle bugs.

- Make the method compatible with DAC and have no ifdefs
- Or, ifdef out the whole method so that you can be sure that it is not used under DAC
- Or, make the method body call `DacNotImpl();`. This one is still not ideal because of the method is still not going to work when called under DAC, but at least the failure is easy to see and diagnose.

Reply:
Thanks for pointing that out, fixed in 14th commit.